src/components/faq.js

```
@@ -23,62 +29,83 @@ const Faqs = ({ faq }) => {
       });
     }
   }, [faq]);
+
   return (
     <>
       {faqSchema && (
         <Head>
           <script
             type="application/ld+json"
-            dangerouslySetInnerHTML={{ __html: DOMPurify.sanitize(JSON.stringify(faqSchema)) }}
+            dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)